source/uwp/Renderer/lib/pch.h

```diff
@@ -40,6 +40,10 @@
 #include "XamlBuilder.h"
 #include "XamlHelpers.h"
 
+namespace render_xaml = ::AdaptiveCards::Rendering::Uwp;
+
+namespace XamlHelpers = render_xaml::XamlHelpers;
```

Comment:
I think we want to keep the winrt stuff in cppwinrtincludes.h, and the non-winrt stuff in pch.h. If we break that convention and move it anyway, it gets messy because of the order of things. The renderer headers in pch need the winrt stuff to compile, and the namespace aliases you mentioned need those headers. If we move them to cppwinrtincludes, the namespaces don't exist yet.